src/drive/web/modules/upload/index.js

```diff
@@ -171,12 +231,26 @@ const getErrors = queue => filterByStatus(queue, FAILED)
 const getQuotaErrors = queue => filterByStatus(queue, QUOTA)
 const getNetworkErrors = queue => filterByStatus(queue, NETWORK)
 const getLoaded = queue => filterByStatus(queue, LOADED)
+const getUploaded = queue => getLoaded(queue).filter(f => !f.isUpdate)
+const getUpdated = queue => getLoaded(queue).filter(f => f.isUpdate)
 
 export const getUploadQueue = state => state[SLUG].queue
 export const getProcessed = state =>
   getUploadQueue(state).filter(f => f.status !== PENDING)
 export const getSuccessful = state => getLoaded(getUploadQueue(state))
 
+export const selectors = {
+  getConflicts,
+  getErrors,
+  getQuotaErrors,
+  getNetworkErrors,
+  getLoaded,
```

Comment:
Do we still want to export `getLoaded`?

Reply:
Good point, it probably better to keep it in here!